arviz/data/base.py

```diff
@@ -256,11 +256,14 @@ def wrapped(self, *args, **kwargs):
 
         return None if _inplace else out
 
-    description = """
-    This method is extended from xarray.Dataset methods. For more info see :meth:`xarray:xarray.Dataset.{method_name}`
+    description = "" if description is None else "\n" + description
+    description_default = """
+    {method_name} method is extended from xarray.Dataset methods.{description}
+    For more info see :meth:`xarray:xarray.Dataset.{method_name}`
```

Comment:
check the breaklines. with numpydoc, docstrings should have a single sentence in the first line (generally goes right after the """ too, not below), then one empty line and then the extended description. Same thing for breaklines between sections.